great_expectations/rule_based_profiler/data_assistant_result/data_assistant_result.py

```diff
@@ -3386,13 +3386,16 @@ def _create_display_chart_for_column_domain_expectation(
             plot_mode=plot_mode,
         )
 
-        return self._chart_column_values(
-            expectation_type=expectation_type,
-            column_dfs=column_dfs,
-            metric_names=metric_names,
-            plot_mode=plot_mode,
-            sequential=sequential,
-        )
+        if len(column_dfs) > 0:
```

Comment:
This bug in plotting was cropping up in various situations when all metrics fail to resolve on a particular domain. I saw it happen while this PR was WIP and this just ensures that the plotting doesn't break in that case.